### DISCUSSION TITLE
Issue: Signed documents print with "Visual Aids" visible when using window.print option

### DISCUSSION THREAD

**Discussion by gitcodehelp** [sheltermanager/asm3#1490] [Q&A / Help]
This relates to an unresolved aspect of Issue/Bug #409.

Issue:
Clients often want a printed copy of their document after signing on the signature pad.
Signed documents will display in the editor/viewer with "Visual Aids" visible by default, and will print as such if using the option: 

Options > Documents > "Printing word processor documents uses hidden iframe and window.print" = checked

Unchecking that option will generate/print an interim preview without aids, but now users must go through extra steps to print anything.

Related Feature Request:
Unless they are designing a template (rare), my users prefer to always have the aids hidden anyway, so my solution has been to hide aids by default in the source code:

File html.py:  def tinymce_header(... visualaids: bool = False ...)

Could you possibly add this as a global option?  Such as:
Options > Documents > checkbox: "Always start word processor with Visual Aids disabled"

(or, eventually, as a per-user option under their User Settings)?

Thank you.

**Comment by bobintetley** [comment]
They can be turned off manually with View->Visual Aids prior to hitting print. 

That said, I don't know why TinyMCE is leaving them on when printing, but we could intercept the print command in TinyMCE and toggle the option off to make sure they never print. I can't think of a scenario where they should? If it's only signed docs that are the issue, we could turn them off by default when viewing/printing signed docs too.

It looks like we already manually turn them off when printing from a separate window with the option you mentioned.

Moved to issue #1491

**Reply by gitcodehelp** [reply]
Yes, it is only signed docs at issue.  When you open a signed doc, the TinyMCE header displays icons for Print and PDF only -- there is no View->Visual Aids menu access.
